test/versioned/mongodb/package.json

```diff
@@ -31,9 +32,25 @@
         "collection-index.tap.js",
         "collection-misc.tap.js",
         "collection-update.tap.js",
+        "cursor-duration.tap.js",
         "cursor.tap.js",
         "db.tap.js"
       ]
+    },
+    {
+      "engines": {
+        "node": ">=10"
+      },
+      "dependencies": {
+        "mongodb": ">=3.3.0"
```

Comment:
I see we are adding >=3.3.0 here. Wonder why it was filtered out. Could be a version in there where things are broken. Suppose we can cross then when we get the minor version testing running again. Will be nice to at least have some recent coverage.

Reply:
yea the tests before my change were only running up to 3.3.0. If you look, the collection-misc and db tests do not work in versions > 3.3.0

So on 3.6.9 these are the failures

`collection-misc.tap.js`

```

            not ok 14 - child should be named Datastore/statement/MongoDB/testCollection/toArray
              ---
              compare: ===
              at:
                line: 102
                column: 17
                file: collection-common.js
              stack: >
                collection-common.js:102:17

                onResult2 (collection-misc.tap.js:32:9)

                DatastoreShim.applySegment (/Users/revans/code/node-newrelic/lib/shim/shim.js:1428:20)

                wrappedCallback (/Users/revans/code/node-newrelic/lib/shim/shim.js:1285:21)

                node_modules/mongodb/lib/utils.js:704:5

                handleCallback (node_modules/mongodb/lib/utils.js:109:55)

                node_modules/mongodb/lib/cursor.js:840:66

                node_modules/mongodb/lib/utils.js:704:5

                node_modules/mongodb/lib/cursor.js:925:9

                AggregationCursor._endSession (node_modules/mongodb/lib/core/cursor.js:397:7)

                node_modules/mongodb/lib/cursor.js:923:12

                maybePromise (node_modules/mongodb/lib/utils.js:692:3)

                AggregationCursor.close (node_modules/mongodb/lib/cursor.js:916:12)

                node_modules/mongodb/lib/cursor.js:840:27

                handleCallback (node_modules/mongodb/lib/core/cursor.js:32:5)

                node_modules/mongodb/lib/core/cursor.js:683:38

                _setCursorNotifiedImpl (node_modules/mongodb/lib/core/cursor.js:696:10)

                setCursorNotified (node_modules/mongodb/lib/core/cursor.js:683:3)

                setCursorDeadAndNotified (node_modules/mongodb/lib/core/cursor.js:676:3)

                nextFunction (node_modules/mongodb/lib/core/cursor.js:830:5)

                AggregationCursor._next (node_modules/mongodb/lib/core/cursor.js:202:5)

                fetchDocs (node_modules/mongodb/lib/cursor.js:834:16)

                node_modules/mongodb/lib/cursor.js:853:11

                handleCallback (node_modules/mongodb/lib/core/cursor.js:32:5)

                nextFunction (node_modules/mongodb/lib/core/cursor.js:867:5)

                node_modules/mongodb/lib/core/cursor.js:743:7

                done (node_modules/mongodb/lib/core/cursor.js:461:7)

                queryCallback (node_modules/mongodb/lib/core/cursor.js:503:20)

                node_modules/mongodb/lib/core/cursor.js:548:9

                node_modules/mongodb/lib/utils.js:704:5

                executeCallback (node_modules/mongodb/lib/operations/execute_operation.js:65:7)

                node_modules/mongodb/lib/operations/command_v2.js:102:9

                operation.cb (node_modules/mongodb/lib/core/connection/pool.js:923:37)

                node_modules/mongodb/lib/core/connection/pool.js:405:18
              source: >2
                              current = current.children[0]
                              t.equal(current.name, segments[i], 'child should be named ' + segments[i])
                ----------------^
                              if (common.MONGO_SEGMENT_RE.test(current.name)) {
                                checkSegmentParams(t, current)
              diff: |
                --- expected
                +++ actual
                @@ -1,1 +1,1 @@
                -Datastore/statement/MongoDB/testCollection/toArray
                +Datastore/statement/MongoDB/$cmd/toArray
              ...
```

`db .tap.js`

```
            not ok 3 - timeout!
              ---
              signal: SIGTERM
              handles:
                - type: Socket
                  events:
                    - end
                    - error
                    - timeout
                    - close
                    - data
              expired: TAP
              test: without transaction
              ...

            1..3
            # failed 1 of 3 tests
        not ok 1 - without transaction # time=59451.917ms
```

I can log a ticket and we can look into them later

Reply:
I logged the ticket around the `collection-misc.tap.js` failure https://github.com/newrelic/node-newrelic/issues/793

Reply:
so the db.tap.js is failing because the callback to `db.addUser` doesn't return the user that was just added just an object that says `{ ok: true }`

Reply:
Filed https://github.com/newrelic/node-newrelic/issues/794 to look into the db.tap.js failures